### DISCUSSION TITLE
METARs with VCBL* codes are not parsing correctly

### DISCUSSION THREAD

**Discussion by ManojAgrawal** [aeharding/metar-taf-parser#47] [Q&A] [answered]
Hello,
METARs with VCBLDU, VCBLSA and VCBLSN such as 
"CYVM 282100Z 36028G36KT 1SM -SN DRSN VCBLSN OVC008 M03/M04 A2935 RMK SN2ST8 LAST STFFD OBS/NXT 291200UTC SLP940" 
are not being parsed correctly as VCBLSN is not being reported back. Is there a fix for this?

thanks,
Manoj

**Comment by aeharding** [comment]
Hi @ManojAgrawal I can confirm this is a bug! Please see https://github.com/aeharding/metar-taf-parser/issues/48

**Comment by aeharding** [comment, marked as answer]
Hi @ManojAgrawal, this should be fixed in version `7.1.1`. Please upgrade metar-taf-parser and let me know if you have any further issues!

Example site with fixed METAR:

https://aeharding.github.io/metar-taf-parser/metar?input=CYVM+282100Z+36028G36KT+1SM+-SN+DRSN+VCBLSN+OVC008+M03%2FM04+A2935+RMK+SN2ST8+LAST+STFFD+OBS%2FNXT+291200UTC+SLP940

**Reply by ManojAgrawal** [reply]
Thanks Alexander!

**Reply by ManojAgrawal** [reply]
Hi Alexander,

I am using a local copy of code from this https://github.com/mivek/python-metar-taf-parser as I get an error while importing the library. How do I copy your code because I don't see any changes in mivek's code.

thanks,
Manoj

**Reply by aeharding** [reply]
I'm not sure I understand, python-metar-taf-parser is a separate project.
